pkg/network/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -213,11 +213,23 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
 					},
 					Gw: defaultGW,
 				}
-				err = netlink.RouteAdd(route)
-				if err != nil {
+				if err := netlink.RouteAdd(route); err != nil {
 					return fmt.Errorf("failed to configure macvlan device: %v", err)
```

Comment:
let's fix up the lame error messages here, for future debugging purposes. `"failed to add route to node IP: %v"`